src/simplewallet/simplewallet.cpp

```diff
@@ -214,7 +214,7 @@ namespace
   const char* USAGE_GET_TX_NOTE("get_tx_note <txid>");
   const char* USAGE_GET_DESCRIPTION("get_description");
   const char* USAGE_SET_DESCRIPTION("set_description [free text note]");
-  const char* USAGE_SIGN("sign <filename>");
+  const char* USAGE_SIGN("sign [<account_index>,<address_index>] <filename>");
```

Comment:
Why move the filename position? Putting the indexes after would mean existing usage remains unchanged (defaults to primary account).

Reply:
Because it's what other commands do. AFAIK existing usage remains unchanged. Did you try it and it changed ?

Reply:
No, I stunk at reading the code last time - too quickly again.. Let me try again.